subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/verification/model/IgnoredKey.java

```diff
@@ -22,7 +22,7 @@ public class IgnoredKey {
     private final String reason;
 
     public IgnoredKey(String keyId, @Nullable String reason) {
-        this.keyId = keyId;
+        this.keyId = keyId.toUpperCase();
```

Comment:
❌ Case-changing functions that are not intended to be locale-sensitive should pass `Locale.ROOT` explicitly.

I don't think it materially matters here, since the input is only ever `[a-f0-9]`, but I think it is better to avoid any possible issues in the future.